README.md

```diff
@@ -10,14 +10,22 @@ For guides on installing, using and configuring CoffeeLint, head over
 To suggest a feature, report a bug, or general discussion, head over
 [here](http://github.com/clutchski/coffeelint/issues/).
 
-## Team
+## :rotating_light: Team :rotating_light:
+
+There are no future plans for CoffeeLint. Only major bugs will be addressed
```

Comment:
I assume you're willing to add maintainers if they volunteered though? (Or transfer ownership or whatever). Maybe that should be mentioned here?